src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -147,7 +147,7 @@ public static Config loadConfig() throws ConfigurationException
         String loaderClass = System.getProperty("cassandra.config.loader");
         ConfigurationLoader loader = loaderClass == null
                                      ? new YamlConfigurationLoader()
-                                   : FBUtilities.<ConfigurationLoader>construct(loaderClass, "configuration loading");
+                                   : FBUtilities.construct(loaderClass, "configuration loading");
```

Comment:
I'm afraid we need this for Java 1.7 compliance. I'm afraid that this is not the first thing in the current codebase for 2.2 that doesn't satisfy [that requirement](https://github.com/apache/cassandra/blob/cassandra-2.2/README.asc#requirements), so I'm opening a ticket to fix other problems with language level compliance.

Reply:
returned, good catch. Considering we don't have (we don't right?) Java 7 CI I didn't think about it :( I will correct it everywhere, good catch, thanks!!!